From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Sun, 30 Apr 2023 17:10:38 +0200
Subject: [PATCH] CMake: Remove `-march=native`

The compiler complains about this:

  error: bad value 'native' for '-march=' switch

However, in the list of supported options, `native` is included - so
there's something funky going on. Leaving this option out lets the game
compile successfully.
---
 SamTFE/Sources/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/SamTFE/Sources/CMakeLists.txt b/SamTFE/Sources/CMakeLists.txt
index a35b48218f74795d553754ef7e5714d6c8755535..44bf38675ac1e943bd4fb1532d26b713c8f9c9bd 100644
--- a/SamTFE/Sources/CMakeLists.txt
+++ b/SamTFE/Sources/CMakeLists.txt
@@ -105,7 +105,6 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
 	add_compile_options(-pipe)
 	add_compile_options(-fPIC)
 	if(NOT PANDORA AND NOT PYRA AND NOT (MACOSX AND CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64"))
-		add_compile_options(-march=native)
 	endif()
 	if(CMAKE_SYSTEM_PROCESSOR MATCHES "^arm.*")
 		if(PYRA)
